tgui/packages/tgui/interfaces/ChemMaster.js

```diff
@@ -332,8 +336,21 @@ const PackagingControls = (props, context) => {
             type: 'vial',
             amount: vialAmount,
             volume: 'auto',
-          })} /> // SKYRAT EDIT HYPOVIALS END
-      )} 
+          })} />
+      )}
+      {!condi && (
+        <PackagingControlsItem
+          label="Smartdarts"
+          amount={dartAmount}
+          amountUnit="darts"
+          sideNote="max 10u"
+          onChangeAmount={(e, value) => setdartAmount(value)}
+          onCreate={() => act('create', {
+            type: 'smartdart',
+            amount: dartAmount,
+            volume: 'auto',
+          })} /> // SKYRAT EDIT MEDICAL END
+      )}
```

Comment:
Yes, this is how they worked before on oldbase.

Reply:
It makes more sense for these to fabricated from the ChemMaster as opposed to being printed because they are only able to be used in syringe guns and are unable to directly deposit their contents into people or containers.